CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Change Log
 
+## 0.0.14 (Jully 2nd, 2020)
+
+- enhancement - Using vulnerability data from Snyk.
+- enhancement - Includes latest updated security vulnerabilities of `maven`, `node` and `pypi` ecosystems, also includes Snyk curated Security Advisories/Vulnerability Unique to Snyk.
+- enhancement - Updated fabric8-analytics-lsp-server to latest version (v[0.2.1](https://www.npmjs.com/package/fabric8-analytics-lsp-server/v/0.2.1)): See [#381](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/381)
+    - Different Underline colors for commonly known vulnerabilities and vulnerability unique to snyk. See [#118](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/118/files#diff-afd7474282d01269197c1d3f05651761R166-R172)
+    - Updated Diagnostic Message: See [#118](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/118)
+        - Number of Known Security Vulnerabilities and Security Advisories for each dependency.
+        - Highest Severity of vulnerabilities for each affected Dependency. (`Low`/`Medium`/`High`/`Critical`)
+        - Recommended version for Dependencies having Known Security Vulnerabilities.
+        - Added Snyk attribution “Powered by Snyk” in the source of the Diagnostic. See [#121](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/121)
```

Comment:
Diagnostic -> diagnostic message.

Reply:
Diagnostic has a separate filed for source, which is the by default part of each diagnostic.